src/styles/index.scss

```diff
@@ -1,3 +1,105 @@
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+h3 {
+  margin: 0;
+}
+
+.cards {
+  box-sizing: border-box;
+  color: #060b35;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to sort CSS properties alphabetically to improve readability and maintainability of the code.